Dockerfile-debian.template

```diff
@@ -20,6 +20,8 @@ RUN set -eux; \
     rm rustup-init; \
     chmod -R a+w $RUSTUP_HOME $CARGO_HOME; \
     \
+    apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false; \
```

Comment:
In Dockerfile-slim, you remove wget, here you remove.. everything? Or what does `apt-get purge` without an argument do?